docs/pages/app.md

```diff
@@ -33,3 +35,54 @@ document.selectedLayers.forEach(layer => console.log(layer.id))
 ### `zoom` number
 
 _Actual Size_ has a zoom level of `1`. To zoom out specify values between `0...1` while using values greater than `1` to zoom in, e.g. `2` for 200%.
+
+## Run a plugin command
+
+**Availability:** _Sketch 55+_
+
+Run a plugin command and pass query parameters to the a JavaScript handler.
+
+```sh
+sketch://plugin/my.plugin.identifier/my.command.identifier
+```
+
+#### Example
+
+The command must implement the `HandleURL` action and have it defined in the plugin manifest in order to be run.
+
+##### `command.js`
+
+```js
+const sketch = require('sketch')
+
+function handleURL(context) {
```

Comment:
a bit tricky here: if using skpm, this will not work, it needs to be `export function handleURL...`

Reply:
Hmm, yeah … I stumbled into this, too. Perhaps I just add a comment above?

```js
// export function handleURL(context) if you're using skpm
```

Reply:
sounds good